src/test_trigrams.py

```diff
@@ -5,12 +5,12 @@
 PATH = os.path.join(os.path.dirname(__file__), 'test.txt')
 
 
-def test_trigrams():
+def test_create_trigrams_dict():
     """Test for conversion to dictionary."""
-    from trigrams import get_txt
-    from trigrams import trigrams
-    assert len(trigrams(get_txt())) == 197 \
-        and trigrams(get_txt())['Holmes she'] == ['is']
+    from trigrams import get_text
+    from trigrams import create_trigrams_dict
```

Comment:
Don't import more than one function from a module in a single test.  Pass through hard-coded values required for your tests.   This will narrow a test to testing only one specific thing.